hera_qm/firstcal_metrics.py

```diff
@@ -406,8 +426,23 @@ def run_metrics(self, std_cut=0.5):
         metrics['frac_JD'] = self.frac_JD
         metrics['std_cut'] = std_cut
         metrics['pol'] = self.pol
+
+        # try to load rotated antenna metric
+        if rotant_json is not None:
+            try:
+                with open(rotant_json) as rot_file:
+                    rmetric = json.load(rot_file)
+                self.rotated_antennas = np.array(rmetric['rotated_antennas']).astype(np.int).tolist()
+                metrics['rot_ants'] = self.rotated_antennas
+            except IOError:
+                print("couldn't load {0}".format(rotant_json))
+                metrics['rot_ants'] = None
+        else:
+            metrics['rot_ants'] = None
```

Comment:
I'm confused. I thought the conclusion from the conversation on this PR was that firstcal_metrics could calculate the `rot_ants` on its own, and there really isn't a need for the json to go between. If all the information is there, I think it's much preferable to not use this intermediate json.

Reply:
Agreed. It's not good enough it just get the delays with fc metrics. We also need to measure the rotations.